README.md

```diff
@@ -86,8 +86,19 @@ sparkcluster.spark.apache.org "prod" deleted
 
 ## Run Spark Pi App on Apache YuniKorn scheduler
 
+If you have not yet done so, follow [YuniKorn docs](https://artifacthub.io/packages/helm/yunikorn/yunikorn#installing-the-chart) to install the latest version: 
```

Comment:
Please use `https://yunikorn.apache.org/docs/#install` instead of `https://artifacthub.io/...`